### DISCUSSION TITLE
MSVC vs MSYS2 installation

### DISCUSSION THREAD

**Discussion by keliburn** [dosbox-staging/dosbox-staging#2649] [General]
When installing DOSBox Staging on Windows (in my case, Windows 11), there is an option to use the MSVC Release binary.  What exactly is different about using that vs the default MSYS2 binary?  Is there any reason to pick one over the other?  On Windows 11, is one or the other option recommended over the other?  It probably isn't a big deal, but I can't find any information on what the ramification on making this choice is.  Thanks!

**Comment by kcgen** [comment]
The MSVC vs MSYS2 builds differ in that the latter supports Ethernet networking (most commonly used/needed in Windows 3.11, see [here](https://github.com/dosbox-staging/dosbox-staging/wiki/Windows-3.1x#native-ip-networking-setup)).

Thanks for this heads up, @keliburn, I've added the above note to the release text, too: https://github.com/dosbox-staging/dosbox-staging/releases/tag/v0.80.1
